en-US/ranking/criteria.md

```diff
@@ -2,117 +2,218 @@
+- Examples of cases where **simplifications to completely accurately following the music should be favoured** include but are not limited to:
+     - Causing an inappropriate difficulty spike relative to the rest of the map
+     - Song can be effectively represented with more consistent and predictable snap
+     - Pattern would be unreasonable to execute with perfect accuracy
+     - Intended difficulty of the map does not match the song intensity
+- **Maps with multiple BPM points should have a constant scrollspeed across them.** This can be achieved by either turning off the option "BPM Affects SV" from the editor or manually countering the scrollspeed effects created by varying BPM using SV points. This does not apply in the rare cases where the intention is to create scrollspeed effects using the song's tempo variance.
+- **Scroll Velocity changes should reflect the music.** Try to avoid changing SV when the music doesn’t call for it. This will generally make your map more difficult to read and harder to enjoy.
+
+## Difficulty Spread
 
 | Length        | Minimum Required Maps |
 | ------------- | :-------------------: |
 | **0:45-2:29** |           2           |
 | **2:30+**     |           1           |
 
-##### Skipping Difficulties
+### Rules
+
+- The standard names for difficulties in a mapset are **Beginner, Easy, Normal, Hard, Insane, and Expert**. Should you choose not to use the standard names, the difficulty names in your set must show clear sign of progression in difficulty consistent with that of the mapset and should be easy to interpret. The highest difficulty of each game mode is always free to have a custom difficulty name.
+     - Standard difficulty name may be added to the end of custom difficulty name in brackets to improve the interpretation of difficulty.
+- You are **not** permitted to skip difficulties for **sets with a song length between 0:45-2:29.** For instance, if you are making a 1 minute set with an Insane, you can either create a Hard or an Expert difficulty. Having a Beginner and an Insane is not allowed. **This rule does not apply to sets with a song length of 2:30 or longer.**
+- **Each game mode is treated separately.** If you are creating a mapset for both 4 Keys and 7 Keys, you must have a difficulty spread for both game modes.
+- **If a mapset has two game modes, each difficulty must be prefaced with either `4K` or `7K`** (Example: `7K Insane`).
+- **If a mapset contains guest difficulties, the guest difficulties creators must be credited in their respective difficulty names** (Example: `Flamingo's Hard`). Mapset's host should not be credited in this fashion.
+- **The number of difficulties created by the host of a mapset must be equal or higher than the largest number of guest difficulties created by a single user.** If a mapper with guest difficulties has contributed to the set with more difficulties than the host, the host should be changed.
```

Comment:
I remember some discussion before about loosening this rule as well.

One might argue that for a spread where a guest maps easy, normal, hard and host maps insane, expert, it would make more sense for the host to stay the same